src/app.js

```diff
@@ -67,6 +68,9 @@ client.on('message', msg => {
       case commandMatches(['help', 'h']):
         help(msg);
         break;
+      case commandMatches('emoji'):
```

Comment:
`commandMatches` checks if any of given command names are exactly what the user has given (ignoring white spaces), so it wouldn't work in your case since you want to be able to accept two arguments unfortunately, so you'll have to make a check of your own or make a new function for commands accepting arguments